docs/install.md

```diff
@@ -66,10 +77,13 @@ NAME           	VERSION	DESCRIPTION
 svc-cat/catalog	0.0.1  	service-catalog API server and controller-manag...
```

Comment:
nit: the version is a constantly moving target (today its 0.1.3).  Should we update the sample output with x.y.z or other indication that it should relate to the current Catalog version?